DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreConfigurationCentreContentDetails.cshtml

```diff
@@ -0,0 +1,110 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model CentreConfigurationViewModel

```

Comment:
Do we need lines 1 and 3 here? Same for the ManagerDetails vc.

Reply:
Ah no I don't think we do, just copied and pasted them and the code cleanup didn't get rid of them again